vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/crawler/transformer/HeadlessCrawlerTableTransformer.java

```diff
@@ -97,4 +99,9 @@ public void setSeedRelativeUrls(Set<String> seedRelativeUrls)
     {
         this.seedRelativeUrls = seedRelativeUrls;
     }
+
+    public void setAllowedExtensions(Set<String> allowedExtensions)
```

Comment:
I think unit tests for `HeadlessCrawlerTableTransformer` should be updated as well to make sure the change is properly integrated